javascript-light-bulb/main.js

```diff
@@ -4,22 +4,22 @@ var $room = document.querySelector('.room');
 var $roomClassList = $room.classList;
 var $buttonContainer = document.querySelector('.button-container');
 var $buttonContainerClassList = $buttonContainer.classList;
-var status = 'light-on';
+var status = true;
```

Comment:
Now this variable name doesn't really convey the purpose. I would change the name to something like `isLightOn`